### PR TITLE
Grd 492 improvements

### PR DESCRIPTION
**Pull request by gavin-peng**

### CODE REVIEW

File: mutect2.wdl [hunk 1]
```diff
@@ -12,6 +19,9 @@ workflow mutect2 {
     File? ponIdx
     File? gnomad
     File? gnomadIdx
+    String reference
+    String gatk
+    String outputFilePrefix
```
Comment (pruzanov) [inline mutect2.wdl]: It needs to be **outputFileNamePrefix**

File: mutect2.wdl [hunk 2]
```diff
@@ -34,12 +44,34 @@ workflow mutect2 {
     }]
```
Comment (pruzanov) [inline mutect2.wdl:51]: Since _gatk_ is a parameter, it probably needs to be removed from the list of dependencies

**Review by pruzanov — CHANGES_REQUESTED** [review]

File: mutect2.wdl (hunk 1 above)
Comment (gavin-peng) [reply mutect2.wdl]: fixed.

File: mutect2.wdl (hunk 2 above)
Comment (gavin-peng) [reply mutect2.wdl:51]: I removed it from dependency and added meta for gatk in wdl, as well as other parameter meta.

**Review by pruzanov — APPROVED** [review]

**Review by pruzanov — APPROVED** [review]

**Pull request merged**